### ISSUE TITLE
Exit code from a script

### ISSUE THREAD

**Issue by feanor12** [dmolina/DaemonMode.jl#22]
Is it possible to return the correct exit code when using runfile()?

Maybe overwriting exit with a return would enable to exit correctly without killing the server.
```
exit(x) = return x
exit(4)
```

**Comment by dmolina** [comment]
Thank you for the idea. However, how do you think it should be consulted the exit code? As the return of runfile() function?

**Comment by feanor12** [comment]
Yes, I think a return value or struct containing the exit code would work.

**Comment by dmolina** [comment]
Now runargs() returns error code if any Exception is run. However, I still have not yet fixed the exit()  problem.

**Comment by feanor12** [comment]
Would it be useful to be able to pass binary objects between the server and the client?
I was thinking maybe the exception(or any other return value) can be passed.
This would require some updates in the server-client protocol though.
Maybe [MsgPack.jl](https://github.com/JuliaIO/MsgPack.jl) or [Serialization](https://docs.julialang.org/en/v1/stdlib/Serialization/#Serialization.serialize) would work?

**Comment by dmolina** [comment]
Now the last version, v0.1.7 allow to run the exit function in the client without any problem. Even more, the result of the exit function is returned by the client (well, 0 or 1).  Finally I have created a local  exit function that returns a specific Exception that is detected by the Daemon for not considering it as a  real Exception. 

Sorry for the delay, but I was working a little allowing threads, that I considered had higher priority.

**Comment by dmolina** [comment]
About the passing of binary objects, I expect to add it in a future remote version (but maybe it could be done in a new package, DaemonRemoteMode.jl?).
